### PR TITLE
chore: fill in gaps for release process

### PR DESCRIPTION
**Pull request by bryceosterhaus**
I took some of the recommendations from this Slack thread, https://liferay.slack.com/archives/C3JBR21HA/p1616066596085500

Let me know if I can make it more clear or add anything else

### CODE REVIEW

**Review by matuzalemsteles — APPROVED** [review]
LGTM! cc @ambrinchaudhary you can probably provide more feedback here

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -278,8 +278,22 @@ git push $REMOTE master --follow-tags
 # Publish to NPM.
 # Make sure you are in the directory of the package you want to publish.
 yarn publish
+
+# Congratulations! You should see successfully published packages.
+#
+# You may also need to do some post-publish administration and close
+# the current Milestone in the github interface,
```
Comment (wincent) [inline CONTRIBUTING.md:285]: github → GitHub if you can be bothered

asking-for-a-friend

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -278,8 +278,22 @@ git push $REMOTE master --follow-tags
...
+#
+# Closing the milestone will trigger a github action to comment on each closed
+# issue in the milestone, notifying that the issue is fixed and publically
```
Comment (wincent) [inline CONTRIBUTING.md:288]: publically → publicly

**Comment by brunofernandezg** [comment]
Thanks Bryce! Love it!

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
Thanks a lot @bryceosterhaus for adding all this. LGTM!

**Pull request merged**